static/js/public/hero-tabpanel.js

```diff
@@ -0,0 +1,211 @@
+class HeroTabpanels {
```

Comment:
If the convention is to use CamelCase, shouldn't this be `HeroTabPanels`?

Reply:
yes, you are right